java/src/jmri/jmrit/beantable/SignalMastTableAction.java

```diff
@@ -3,16 +3,29 @@
 import java.awt.Component;
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
+import java.util.Hashtable;
```

Comment:
Where is this used in this file? I can't find it. Please don't include imports that are not used. (Your IDE can easily flag these and may even be able to automatically remove unused imports.)